modules/nw-ovn-kubernetes-migration-about-live.adoc

```diff
@@ -0,0 +1,166 @@
+
+== Prerequisites for live migration
+
+The following prerequisites must be met before migrating to the OVN-Kubernetes network plugin from the OpenShift SDN plugin:
+
+* A cluster has been configured with the OpenShift SDN CNI network plugin in the network policy isolation mode.
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have created a recent backup of the etcd database is available.
+
+* The cluster is in a known good state without any errors.
+
+* On all cloud platforms after updating software, a security group rule must be in place to allow UDP packets on port `6081` for all nodes.
+
+* Cluster administrators must disable the following custom resources:
++
+** Egress Router
+** Egress IP
+** Egress Firewalls
+** Multicast
+
+== Checking Egress and multicast custom resources
+
+The following custom resources must be disabled before performing a live migration of OpenShift-SDN to OVN-Kubernetes:
+
+* Egress Router
+* Egress IP
+* Egress Firewalls
+* Multicast
+
+You can check the status of these custom resources (CRs) with the following procedure.
+
+.Procedure 
+
+. From the OpenShift Container Platform web console, navigate to *Observe* -> *Metrics*.
+
+. In the *Expression* box, add the following queries, and then click *Add query*:
++
+[source,promql]
++
+----
+network_attachment_definition_instances{networks="egress-router"} > 0
+----
++
+----
+sdn_controller_num_egress_ips > 0
+----
++
+----
+sdn_controller_num_egress_firewalls > 0
+----
++
+----
+sdn_controller_num_multicast_enabled_namespaces > 0
+----
+
+. Click *Run queries*.
++
+If any datapoints are found, you should not proceed with the OpenShift SDN to OVN-Kubernetes live migration procedure. Instead you should use the following information to disable these CRs:
+
+* link:https://docs.openshift.com/container-platform/4.14/networking/openshift_sdn/disabling-multicast.html[Disabling multicast for a project]
+* link:https://docs.openshift.com/container-platform/4.14/networking/openshift_sdn/removing-egress-firewall.html[Removing an egress firewall from a project]
+* link:https://access.redhat.com/solutions/4854561[Removing an egress IP address]
+* To remove an egress router, delete the pod. 
+
+= Using the live migration method to migrate from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin
+
+Administrators should ensure that all prerequisites have been met before beginning the migration process. Afterwards, the following procedure can be used to initiate the live migration from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin.
+
+.Procedure
+
+. Enter the following command to add the `unsupported-red-hat-internal-testing` annotation to the cluster-level network configuration:
```

Comment:
@martinkennelly Was it added when I was on PTO?

Reply:
I didnt add it and I didnt see anyone else add it. I dont know where this annotation came from.

Reply:
that's the annotation required by SD.
Currently there is a webhook which will not allow for live-migration to start unless this annotation is set, this was done to ensure users do not try out live migration before we conclude it all of our testing.

Reply:
See thread: https://redhat-internal.slack.com/archives/C0590SMCUKV/p1707835117080859